code/modules/awaymissions/mission_code/ruins/deepstorage.dm

```diff
@@ -0,0 +1,380 @@
+
+/obj/item/paper/fluff/ruins/deepstorage/log2
+	name = "to my love"
+	info = {"Everything went batshit insane, at first we all thought it was a terrorist attack or something...
+	All floors were put into lockdown, then shortly after all communication across the facility went black.<br>
+	<br>
+	Guards never told us anything about what's going on but the fear in their eyes was all there to be seen.<br>
+	<br>
+	Our colleagues, friends and other folks started arguing, hurting then killing each other in time. Me and four other managed to isolate ourselves in cafeteria.<br>
+	<br>
+	We have enough supply to last a few weeks here, should be enough until reinforcements arrive, right?"<br>
+	<br>
+	If it won't, then god forsake what might happen to us... I can't wait to see you again.<br>
+	<br>
+	Sincerely yours..."}
+
+/obj/item/paper/fluff/ruins/deepstorage/log3
+	name = "quartermaster's personal log"
+	info = {"<b>(beginning of record...)</b><br>
+	<br>
+	Day 5 since the lockdown, we can't communicate with other floors which is quite a nuisance. We were already behind 13 deliveries.
+	At these very times i miss my time back at- <br>
+	<br>
+	<b>(end of record.)</b><br>
+	<br>
+	<b>(beginning of record...)</b><br>
+	<br>
+	Day 5, since the lockdown. More people are getting sick. I heard the body count was so high they had to start cremating them.
+	The more away they are the better. <br>
+	<br>
+	<b>(end of record.)</b><br>
+	<br>
+	<b>(beginning of record...)</b><br>
+	<br>
+	Day... 5, since the-. God, i just checked my previous entries and there was like, 14 in total? In a big portion of it i start with 'day 5 of this, of that'.
+	Weird part is, i don't recall any of those records. It must be my insolent staff, right. They're yearning for some bad words in their employee records
+	and that they shall receive...<br>
+	<br>
+	<b>(end of record.)</b><br>"}
+
+/obj/item/paper/fluff/ruins/deepstorage/log4
+	name = "crematorium report"
+	info = {"We burnt so many... There's no end to this. It's been days and we're still burning them."}
+
+
+/obj/item/paper/fluff/ruins/deepstorage/log5
+	name = "subject: my concerns"
+	info = {"First of all i appreciate the good initative on cutting off comms, although i am sure there are some rats listening the private frequency.
+	Unless you desire my floor to embrace a chaos, i suggest you to keep quiet in there.<br>
+	<br>
+	Off to the matter, we have been working on the sample tissue you have sent non-stop for days by now. In my proficiency one might assume they saw all
+	bio-abominations there is to be seen. I never seen anything like this, because it's not a virus or disease or anything that you claim to be.<br>
+	<br>
+	Yes, it's contagious and i don't think it's limited with physical contact. We have been burning the corpses, there's still more to burn.
+	This will not end well, i advise evacuating workers to a more suitable quarantine zone before things goes out of hand."}
```

Comment:
```suggestion
	info = {"First of all, I appreciate your initiative in cutting off comms, although I'm sure there are some rats listening on the private frequency.
	Unless you want a riot on my floor, I suggest keeping quiet in there.<br>
	<br>
	Getting down to business: we have been working on the sample tissues you've been sending non-stop. In my expert opinion, I don't think there's a single
	bio-abomination we haven't analyzed. I've never seen anything like this. It's nothing like a virus or disease you might be familiar with.<br>
	<br>
	Yes, it's contagious, and I don't think it's limited to physical contact. We have been burning the corpses; there's still more to burn.
	This will not end well. I advise evacuating workers to a more suitable quarantine zone before things get out of hand."}
```